File: backend/src/basico/002-ejercicios/001-mayor_de_dos_numeros/higher_number.c
#include <stdio.h>

int mayor(int a, int b);

int main() {
    int num1 = 3, num2 = 5;
    printf("%d\n", mayor(num1, num2));

    return 0;
}

int mayor(int a, int b) {
    if (a > b) { return a; } else { return b; }
}

File: backend/src/basico/002-ejercicios/001-mayor_de_dos_numeros/higher_number.go

package main

//import "math"

func mayor(a int, b int) (int) {
    if a >= b {
        return a
    } else {
        return b
    }
}

func main() {
    num1, num2 := 3, 5  // Asignación múltiple
    println(mayor(num1, num2))

    // También así (requiere descomentar import "math":
    //respuesta := int( math.Max(float64(num1), float64(num2)) )
    //println(respuesta)
}

/* Fuentes:
https://golang.org/pkg/math/#Max
https://stackoverflow.com/questions/19230191/convert-an-integer-to-a-float-number-in-golang
*/
